modules/backing-up-red-hat-quay.adoc

```diff
@@ -69,21 +76,54 @@ All information under the `data` property should remain the same.
 $ oc get secret -n <quay-namespace>  $(oc get quayregistry <quay-registry-name> -n <quay-namespace>  -o jsonpath='{.spec.configBundleSecret}') -o yaml > config-bundle.yaml
 ----
 
-. Scale down the Quay the Quay Operator:
+=== Scale down your {productname} deployment
+
+[NOTE]
+====
+This step is needed to create a consistent backup of the state of your {productname} deployment. Do not omit this step, including in setups where Postgres databases and/or S3-compatible object storage are provided by external services (unmanaged by the operator).
+====
+
+. *For operator version 3.7 and newer*: scale down the {productname} deployment by disabling auto scaling and overriding the replica count for Quay, Mirror workers and Clair (if managed). Your `QuayRegistry` resource should look similar to the following:
 +
-[source,terminal]
-----
-$  oc scale --replicas=0 deployment $(oc get deployment -n <quay-operator-namespace> |awk '/^quay-operator/ {print $1}') -n <quay-operator-namespace>
+[source,yaml]
 ----
+apiVersion: quay.redhat.com/v1
+kind: QuayRegistry
+metadata:
+  name: registry
+  namespace: ns
+spec:
+  components:
+    …
+    - kind: horizontalpodautoscaler
+      managed: false <1>
+    - kind: quay
+      managed: true
+      overrides: <2>
+        replicas: 0
+    - kind: clair
+      managed: true
+      overrides:
+        replicas: 0
+    - kind: mirror
+      managed: true
+      overrides:
+        replicas: 0
+    …
+----
+<1> disable auto scaling of Quay, Clair and Mirroring workers
+<2> set the replica count to 0 for components accessing the database and objectstorage
 
-. Scale down the Quay namespace:
+. *For operator version 3.6 and earlier*: scale down the {productname} deployment by scaling down the {productname} operator first and then the managed {productname} resources:
 +
 [source,terminal]
 ----
+$  oc scale --replicas=0 deployment $(oc get deployment -n <quay-operator-namespace> |awk '/^quay-operator/ {print $1}') -n <quay-operator-namespace>
+
 $ oc scale --replicas=0 deployment $(oc get deployment -n <quay-namespace> -l quay-component=quay -o jsonpath='{.items[0].metadata.name}') -n <quay-namespace>
 ----
```

Comment:
In keeping with 3.7,  for 3.6 and earlier,  we also need to scale down Quay, Mirror workers and Clair (if managed). The whole commands should be 
```
$ oc scale --replicas=0 deployment $(oc get deployment -n <quay-namespace>|awk '/^quay-operator/ {print $1}') -n <quay-namespace>
$ oc scale --replicas=0 deployment $(oc get deployment -n <quay-namespace>|awk '/quay-app/ {print $1}') -n <quay-namespace>
$ oc scale --replicas=0 deployment $(oc get deployment -n <quay-namespace>|awk '/quay-mirror/ {print $1}') -n <quay-namespace>
$ oc scale --replicas=0 deployment $(oc get deployment -n <quay-namespace>|awk '/clair-app/ {print $1}') -n <quay-namespace>
```
I found the labels in deployment was changed in different quay versions, so it is better to use `awk` to filter out the deployment name. So I updated the command at the same time.

Reply:
Thanks, that's great.